crates/core/tedge_mapper/src/core/mapper.rs

```diff
@@ -171,25 +171,13 @@ async fn process_inotify_and_mqtt_messages(
                         }
                     }
                 }
-                event = inotify_events.next() => {
-                    match event {
-                        Some(ev) => {
-                            match ev {
-                                Ok(ev_string) => {
-
-                                            match  process_inotify_events(dir.clone(), ev_string) {
-                                                Ok(Some(discovered_ops)) => {
-                                                    let _ = mapper.output.send(mapper.converter.process_operation_update_message(discovered_ops)).await;
-                                                }
-                                                Ok(None) => {}
-                                                Err(e) => {eprintln!("Processing inotify event failed due to {}", e);}
-                                            }
-
-
-                                } Err(e) => {eprintln!("Failed to extract event {}", e);}
-                            }
+                Some(Ok(event)) = inotify_events.next() => {
```

Comment:
It is true that it is not there anymore. I filtered for `Some(Ok(event))` and there is no more `Some(Err(error))`. Maybe I should add that logic back so that errors are clearer to us and users